packages/website/src/components/RecruitmentModal/RecruitmentModal.tsx

```diff
@@ -24,6 +26,8 @@ interface RecruitmentModalProps {
 export const RecruitmentModal = ({ isOpen, config, onClose, modalType }: RecruitmentModalProps) => {
   const modalContentPaddings = useBreakpointValue({ base: '32px 16px 16px 32px', md: '56px 64px 64px 64px' } as const);
   const modalBodyMarginBottom = useBreakpointValue({ base: '16px', lg: '32px' } as const);
+  const modalCloseIconSize = useBreakpointValue({ base: 'sm', md: 'lg' } as const);
```

Comment:
🤔 If I remember correctly, all clickable elements should be at least 40x40px. Small icon button has 32x32px